lib/pages/general/item/item.dart

```diff
@@ -471,6 +483,27 @@ class ItemPage extends StatelessWidget {
     );
   }
 
+  Widget _buildFlagDetails(BuildContext context, List<String> displayFlags) {
+
+    return Wrap(
+        alignment: WrapAlignment.spaceEvenly,
+        spacing: 16.0,
+        runSpacing: 4.0,
+        children:
+          displayFlags.map((f) => Chip(
+            backgroundColor: Theme.of(context).brightness == Brightness.light ? Colors.black12 : Colors.white12,
+            label: Text(
+              typeToName(f),
+              style: Theme.of(context).textTheme.bodyText1.copyWith(
+                  color: Theme.of(context).brightness == Brightness.light ? Colors.black : Colors.white,
+              ),
+            ),
+          ))
+        .toList(),
+    );
```

Comment:
I'd change this function to something like this:
```
return Padding(
      padding: EdgeInsets.all(4.0),
      child: Wrap(
        alignment: WrapAlignment.spaceEvenly,
        spacing: 8.0,
        runSpacing: 8.0,
        children:
          displayFlags.map((f) => Chip(
            materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
            backgroundColor: Theme.of(context).brightness == Brightness.light ? Theme.of(context).accentColor : Colors.white12,
            label: Text(
              typeToName(f),
              style: Theme.of(context).textTheme.bodyText1.copyWith(
                color: Colors.white,
              ),
            ),
          ))
        .toList(),
      ),
    );
```
I've tweaked the padding a bit, so it doesn't take up as much space. And i've changed the color of the chips to the accent color in light theme, as chips should usually represent the accent color.

Reply:
I based the original on the Chip() code you referenced, but this is much nicer.